Meta/run.sh

```diff
@@ -32,7 +32,11 @@ fi
 # we need to use 64-bit qemu
 if [ "$(uname)" = "Darwin" ] && [ "$(uname -m)" = "x86_64" ]; then
```

Comment:
If you replace `[ "$(uname -m)" = "x86_64" ]` with  this ` ( [ "$(uname -m)" = "x86_64" ] || [ "$(uname -m)" = "arm64" ] )` the code will also work for Macs with M1

Reply:
Why not just remove the (uname -m) entirely then?